### ISSUE TITLE
workbench.action.toggleMenuBar toggles between `toggle` and `default`

### ISSUE THREAD

**Issue by hshin23** [Microsoft/vscode#41002]
I searched the keywords on the open issues and there are numerous issues targeting this.

I'm not 100% sure as to why `workbench.action.toggleMenuBar` switches between `toggle` and `default`.

Because when `menuBarVisibility` is `toggle` this allows menuBar to toggle by pressing `alt` key.

I'm not sure if there is a hidden issue with Electron or not, but making it switch between `hidden` and `visible` (or `default`) works fine with no issue in both windowed mode and Zen mode.

TL;DR;
I'm wondering if it is desired to have the `menuBarVisibility` to toggle between `hidden` and `visible` instead of how it's being done currently (between `toggle` and `default`).

**Comment by bpasero** [comment]
Yeah I would say that the behaviour is OK given we had this in for a long time now and not getting complaints so far. I fear changing this would result in the opposite: people asking why the behaviour was suddenly changed. Closing as designed.
